### DISCUSSION TITLE
Selectable Background

### DISCUSSION THREAD

**Discussion by kanarelo** [pikasojs/pikaso#21] [Ideas]
I don't know if this qualifies as an issue or a feature request.

```typescript 
this.image = new ImageModel(board, image, {
  selectable: false,
  history: false
})

this.overlay = new RectModel(board, overlay, {
  selectable: false,
  history: false
})
```

Above is inside the Background constructor. Is there a way we can have an option of having the background image selectable for easy change from the canvas editor?

**Comment by raminious** [comment]
@kanarelo hmm makes sense. Even though I think it should not be selectable at the start, there should be an option to make it selectable. this is a feature request. can you please create an issue for it?

**Reply by kanarelo** [reply]
Sure!

**Comment by raminious** [comment]
addressed in https://github.com/pikasojs/pikaso/issues/22#issuecomment-942689438
